scripts/get-env-venue-owners.ts

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env node -r esm -r ts-node/register
+
+import fs from "fs";
+import { resolve } from "path";
+
+import admin from "firebase-admin";
+
+import {
+  initFirebaseAdminApp,
+  makeScriptUsage,
+  parseCredentialFile,
+} from "./lib/helpers";
+
+const usage = makeScriptUsage({
+  description: "Print venue owners' email addresses.",
+  usageParams: "[CREDENTIAL_PATH] filterArray",
+  exampleParams: "[theMatchingAccountServiceKey.json] gmail.com",
+});
+
+const [credentialPath, ...filterArray] = process.argv.slice(2);
+
+if (!credentialPath) {
+  usage();
+}
+
+const { project_id: projectId } = parseCredentialFile(credentialPath);
+
+if (!projectId) {
+  console.error("Credential file has no project_id:", credentialPath);
+  process.exit(1);
+}
+
+initFirebaseAdminApp(projectId, {
+  credentialPath: credentialPath
+    ? resolve(__dirname, credentialPath)
+    : undefined,
+});
+
+(async () => {
+  const { users, pageToken } = await admin.auth().listUsers(1000);
+
+  const allUsers = [...users];
+  let nextPageToken = pageToken;
+
+  while (nextPageToken) {
+    const { users, pageToken } = await admin
+      .auth()
+      .listUsers(1000, nextPageToken);
+    allUsers.push(...users);
+    nextPageToken = pageToken;
+  }
+
+  // @debt This function will currently load all venues in firebase into memory.. not very efficient
+  const firestoreVenues = await admin.firestore().collection("venues").get();
+  const venues = firestoreVenues.docs;
+
+  // clear file
+  fs.writeFileSync("./venueOwners.csv", "");
```

Comment:
let's rename the file we generate not to mix up the results. Let it be (or similar)
```
project-id-venue-owners.csv
```

for example
```
sparkle-burn-venue-owners.csv
```

and make it a const, please, to reuse later and to change it in one place, if needed in the future.

Reply:
fixed f0ac60e